main.tf

```
@@ -1,6 +1,7 @@
 terraform {
   required_providers {
-    apko = { source = "chainguard-dev/apko" }
+    apko       = { source = "chainguard-dev/apko" }
+    chainguard = { source = "chainguard-dev/chainguard" }
```

is this actually needed?
```suggestion
    apko = { source = "chainguard-dev/apko" }
```